episodes/03-working-with-MARC-files.md

```diff
@@ -173,11 +181,19 @@ If you change your preferences for the MarcEditor, the tool used to work with MA
 ## Reset settings for the MarcEditor in Preferences
 
 1. Click Edit → Preferences
-2. Select "MarcEditor" in the Preferences window in the left hand pane
-3. In the right hand pane, select Set Defaults for either font or font size
+2. Select "MarcEditor" in the Preferences window in the left-hand pane
+3. In the right-hand pane, select Set Defaults for either font or font size
 4. Click Ok
   
 
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::  callout
+
+## Behavior when you reset your preferences and other popup windows
+
+When you click Ok to reset preferences, there is a chance that the popup window for Preferences and/or even MarcEdit main menu will close. If the MarcEdit main menu closes, sometimes it will reopen. If it does not, then click on MarcEdit again on your desktop to reopen the application. It should be noted that not all popup windows will close automatically such as Find, Replace, or the MarcValidator. However, the Select Records for Edit temporary editor will close if you click Save. These idiosyncracies for each will be covered in their upcoming episodes.
```

Comment:
This doesn't happen for me when I save my preferences, only the preference pop up closes - in your testing does the MarcEdit application close for this action?. It may be important to note, though, that in order for the preference changes to display, you will need to close and reopen the MarcEditor, which would require saving your file if you haven't done so.

Reply:
Add callout about preferences needing to be applied by restarting the MarcEditor.